test/specs/lib/keyboardHandlerFilter-test.ts

```diff
@@ -6,35 +6,13 @@ const getEventArg = (
   ctrlKey?: boolean,
   metaKey?: boolean,
   shiftKey?: boolean,
-): React.KeyboardEvent => {
+): any => {
```

Comment:
Is there any chance to avoid `any`?

Reply:
@sophieH29 ?

Reply:
unfortunately no. The method which is tested is using original typescript type ```KeyboardEvent``` (we don't want to use React.KeyboardEvent, to not bind it to any framework.), which kind not possible to create an instance of it as we want for testing purposes. The only thing we want to test that object is correctly filtered when it has different combinations of ctrl, alt, shift, meta set to the object.
@layershifter if you have any ideas on how to avoid it, feel free to share.

Reply:
will leave any for now